mautic/mautic
# Fix SQL issue when creating a backup during install

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [Y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #8457 



#### Description:

The backup step didn't include the options when creating the backed up index, which is necessary for e.g. specifying the length of a text or blob. This PR ensures all the options are passed along.



#### Steps to test this PR:


1. install a vanilla mautic a first time
2. remove the `config/local.php` file
3. install it again, and ensure the backup tables option is on
4. an error is thrown about the `bak_page_hits` index
5. apply the patch
2. remove the `config/local.php` file
3. install it again, and ensure the backup tables option is on
4. no error is thrown, and the index in the `bak_page_hits` table exists




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10747"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
